server/utils/telemetry/rudder.go

```diff
@@ -0,0 +1,38 @@
+package telemetry
+
+import rudder "github.com/rudderlabs/analytics-go"
+
+const (
+	rudderDataPlaneURL = ""
+	rudderWriteKey     = ""
+)
+
+func NewRudderClient() (Client, error) {
+	client, err := rudder.NewWithConfig(rudderWriteKey, rudderDataPlaneURL, rudder.Config{})
```

Comment:
0/5 Both of this function and `NewRudderClientWithCredentials` are very small, but couldn't this function call `NewRudderClientWithCredentials` to simplify? Maybe also match the argument order passed to `NewWithConfig`?